dist/threads/threads.xs

```diff
@@ -193,9 +193,6 @@ S_ithread_set(pTHX_ ithread *thread)
 {
     dMY_CXT;
     MY_CXT.context = thread;
-#ifdef PERL_SET_NON_tTHX_CONTEXT
-    PERL_SET_NON_tTHX_CONTEXT(thread->interp);
-#endif
 }
```

Comment:
Will a change to this file necessitate an increment to `$VERSION` in `threads.pm`?

Reply:
It probably will unless it gets included in this month's development release.